concepts/sepsis3/blood-culture-icu-admit.sql

```diff
@@ -0,0 +1,24 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.blood_culture_icu_admit`;
+CREATE TABLE `physionet-data.mimic_derived.blood_culture_icu_admit` AS
```

Comment:
hmm ideally I'd like to not have this table since it doesn't seem to give much more information than the blood culture table. also it's not limited to blood cultures so the name is a bit of a misnomer :)

Reply:
I saw this table was used in the sepsis3 queries as part of determining sepsis3 criteria. Does this mean we want to replace this table with something else? I was under the impression that sepsis3 requires a sampling of bodily fluids (such as a blood culture).